wfe/wfe.go

```diff
@@ -1684,7 +1684,11 @@ func (wfe *WebFrontEndImpl) KeyRollover(ctx context.Context, logEvent *web.Reque
 }
 
 func (wfe *WebFrontEndImpl) deactivateRegistration(ctx context.Context, reg core.Registration, response http.ResponseWriter, request *http.Request, logEvent *web.RequestEvent) {
-	err := wfe.RA.DeactivateRegistration(ctx, reg)
+	regPb, err := grpc.RegistrationToPB(reg)
+	if err != nil {
+		wfe.sendError(response, logEvent, web.ProblemDetailsForError(err, "Error deactivating registration"), err)
+	}
```

Comment:
If we reduce the checks in ra.go to just "isn't nil, has an ID, has status Valid", then it would be trivial to manually construct a registration proto with just those attributes here, instead of using the full "convert a registration to a PB" method.